src/core/dbcsr_config.F

```diff
@@ -607,7 +625,12 @@ SUBROUTINE dbcsr_print_config(unit_nr)
          END IF
       END BLOCK
 
-      IF (has_acc) THEN
+      IF (dbcsr_cfg%turn_off_acc%val) THEN
+         WRITE (UNIT=unit_nr, FMT='(1X,A,T81,A4)') &
+            "DBCSR| ACC is turned off: only CPU is used", dbcsr_cfg%turn_off_acc%print_source()
+      END IF
+
+      IF (use_acc()) THEN
```

Comment:
We should keep has_acc here, this is because people compiled for the GPU (even if they are not running on it).